src/main/java/roomescape/theme/ThemeService.java

```diff
@@ -1,23 +1,30 @@
 package roomescape.theme;
 
 import org.springframework.stereotype.Service;
+import roomescape.entities.Theme;
 import roomescape.repositories.ThemeRepository;
 import roomescape.theme.data.ThemeAddRequestDto;
 
+import java.util.List;
+
 @Service
 public class ThemeService {
-
     final private ThemeRepository themeRepository;
 
     public ThemeService(ThemeRepository themeRepository) {
         this.themeRepository = themeRepository;
     }
 
-    public void addTheme(ThemeAddRequestDto addRequestDto){
+    public List<Theme> findAllThemes() {
+        return themeRepository.findAllThemes();
     }
 
-    public void deleteTheme(Long id){
+    public Theme save(ThemeAddRequestDto themeAddRequestDto){
+        Theme theme = new Theme(themeAddRequestDto.getName(), themeAddRequestDto.getDescription(), themeAddRequestDto.getThumbnail());
```

Comment:
```suggestion
        Theme theme = new Theme(
            themeAddRequestDto.getName(),
            themeAddRequestDto.getDescription(),
            themeAddRequestDto.getThumbnail());
```

진영님은 가로 줄 제한을 몇으로 두시나요?
intelliJ 화면에 있는 회색선 안에 코드가 담길 수 있게 줄바꿈을 해보면 좋을 것 같아요.
가로로 길어지는 것보단 세로로 길어지는게 그나마 가독성에 도움이 되더라구요 :)

(다른 파일도 한 번 확인해주세요)